docs/resources/aws_redshift_cluster_parameter_groups.md

```diff
@@ -0,0 +1,70 @@
+---
+title: About the aws_redshift_cluster_parameter_groups Resource
+platform: aws
+---
+
+# aws\_redshift\_cluster\_parameter\_groups
+
+Use the `aws_redshift_cluster_parameter_groups` InSpec audit resource to test properties of some or all AWS Redshift Cluster Parameter Group.
+
+## Syntax
+
+Verify that a Parameter group name exists.
+
+    describe aws_redshift_cluster_parameter_groups do
+      it { should exist }
+    end
+
+An `aws_redshift_cluster_parameter_groups` resource block uses an optional filter to select a group of Redshift Cluster Parameter Groups and then test that group.
+
+## Parameters
+
+This resource does not expect any parameters.
+
+See the [AWS documentation on Redshift Cluster Parameter Group](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-redshift-clusterparametergroup.html) for additional information.
+
+## Properties
+
+| Property | Description|
+| --- | --- |
+| parameter_group_names | The name of a specific parameter group for which to return details. By default, details about all parameter groups and the default parameter group are returned. |
+| parameter_group_families | The family of the parameter group. |
+| descriptions | The description of the parameter group. |
+| tags | The tags of the parameter group. |
+
+## Examples
+
+##### Ensure a Redshift Cluster Parameter Group has parameter groups.
```

Comment:
Is this just testing if at least one parameter group exists?

Reply:
yes